main/webapp/modules/core/scripts/dialogs/expression-preview-dialog.js

```diff
@@ -186,8 +183,7 @@ ExpressionPreviewDialog.Widget.prototype._renderHelpTab = function() {
 };
 
 ExpressionPreviewDialog.Widget.prototype._renderHelp = function(data) {
-    var elmt = this._elmts.expressionPreviewHelpTabBody.empty().width(this._tabContentWidth);
-    
+    var elmt = this._elmts.expressionPreviewHelpTabBody.empty().width("700px");
```

Comment:
Why replace this variable by a hard-coded value?

Reply:
I replaced that variable as preview tab containers width is variable now where the preview, history, and starred tabs contained resize when the frame resize.
It is just the help container I set its width with a hard-coded value showing help in a suitable readable way.
would you like me to modify it ?

Reply:
Hmm, I am not sure I understand. It is not clear to me why we would not want the Help tab to expand to fit the dialog width like other tabs. In your third screenshot I find it not very nice to have the scroll bar in the middle of the screen with plenty of white space next to it. But maybe I have missed something?

Reply:
I agree with you, I tried to manipulate it as the other tabs but there is a script causes the help tab to occupy 100% of the browser screen width expanding the frame too in undesired way.
I will modify this script so that help tab is consistent with the other tabs